addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/thermostat/NAPlugHandler.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.netatmo.handler.thermostat;
+
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.openhab.binding.netatmo.handler.NetatmoDeviceHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.swagger.client.model.NADeviceListBody;
+
+/**
+ * {@link NAPlugHandler} is the class used to handle the plug
+ * device of a thermostat set
+ *
+ * @author Gaël L'hopital - Initial contribution OH2 version
+ *
+ */
+public class NAPlugHandler extends NetatmoDeviceHandler {
+    private static Logger logger = LoggerFactory.getLogger(NAPlugHandler.class);
+
+    public NAPlugHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    protected void updateChannels() {
+        try {
+            NADeviceListBody deviceList = bridgeHandler.getThermostatApi().devicelist(actualApp, getId(), false)
+                    .getBody();
+            device = deviceList.getDevices().get(0);
+
+            super.updateChannels();
+        } catch (Exception e) {
+            logger.error(e.getMessage());
```

Comment:
don't log an error, better provide the error message within the Thing status as this will also be shown in the UI.